### PR TITLE
Added moar tests!

### PR DESCRIPTION
**Pull request by FinalAngel**
Reaching the sky

### CODE REVIEW

File: tests/requirements.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
-# requirements from setup.py
-django-filer>=1.2.4
-djangocms-attributes-field>=0.4.0
-# other requirements
 djangocms-helper
 tox
 coverage
+isort
+flake8
+# override "pyflakes<2.1" from djangocms-helper
```
Comment (kinkerl) [inline tests/requirements.txt]: ?? this should not work if it is a normal requirement? why is this not a conflict?

File: tests/test_plugins.py [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+        self.page.publish(self.language)
+        self.assertEqual(folder.audio_folder.name, "test_folder")
+
+        with self.login_user_context(self.superuser):
+            response = self.client.get(request_url)
+
+        self.assertContains(response, "No matching audio files were found in the specified folder.")
+
+        # cleanup
+        audio_file.delete()
```
Comment (kinkerl) [inline tests/test_plugins.py]: should happen in the tear down

File: tests/requirements.txt (hunk 1 above)
Comment (FinalAngel) [reply tests/requirements.txt]: in this case its installed via pip install -r tests/requirements.txt, it throws a warning but installs it nonetheless

File: tests/test_plugins.py (hunk 2 above)
Comment (FinalAngel) [reply tests/test_plugins.py]: Did that on purpose, don't want to create a second file for each run and then delete it, for double checking I've got the cleanup lines:

https://github.com/divio/djangocms-audio/pull/20/files#diff-327b130c82fa4e83e97f6c1a50a0da1cR32

**Pull request merged**